t/t5601-clone.sh

```diff
@@ -68,6 +68,14 @@ test_expect_success 'clone respects GIT_WORK_TREE' '
 
 '
 
+test_expect_success MINGW,CASE_INSENSITIVE_FS 'core.worktree is not added due to path case' '
+
+	mkdir CAPITALCASE &&
+	lower_path=$( cygpath -w "$(pwd)/CAPITALCASE" | tr A-Z a-z ) &&
+	git clone src "$lower_path" &&
```

Comment:
It's actually the lower _case_ path. But I think `lower` would be plenty sufficient.

Reply:
Changed to `lower` as requested.